MathLib/LinAlg/Dense/DenseVector.h

```diff
@@ -75,6 +75,14 @@ class DenseVector : public std::valarray<T>
 		}
 	}
 
+	/// get entry values to an array
+	void getValues(T u[])
```

Comment:
Yes, it is the reason. It will be changed later with a possible change in ``PropertyVector``.